onnxruntime/core/providers/webnn/builders/impl/squeeze_unsqueeze_op_builder.cc

```diff
@@ -58,7 +58,7 @@ Status SqueezeUnsqueezeOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_buil
   std::vector<int32_t> axes_data;
   auto rank = input_rank;
 
-  if (node.SinceVersion() >= 13 && input_defs.size() > 1) {
+  if (node.SinceVersion() >= 13 && input_defs.size() > 1 && !GetTensorName(input_defs, 1).empty()) {
```

Comment:
Consider just putting this inside `GetTensorName` rather than repeating the size check every time, which would be less brittle.

Reply:
> Consider just putting this inside GetTensorName rather than repeating the size check every time, which would be less brittle.

Oops, I forget I already did size check in the `GetTensorName`, remove the duplicated ones.

Reply:
> does GetTensorName return a temporary std::sting?

Some other codes need to get the tensor name.

Reply:
Okie dokie. If we have more cases in the future, it may be worth keeping `GetTensorName` and adding another helper, since it's easier to read `if (condition)` rather than if (!condition.reverse_condition())`. LGTM.